cql3/expr/expression.hh

```diff
@@ -218,10 +217,10 @@ enum class comparison_order : char {
 struct binary_operator {
     expression lhs;
     oper_t op;
-    ::shared_ptr<term> rhs;
+    expression rhs;
```

Comment:
search_and_replace() and friends need to recurse into rhs.

Reply:
There was a comment explaining that it only works on LHS, although that assumes that people read comments.